src/main/java/com/digitax/model/User.java

```diff
@@ -76,11 +88,14 @@ public User() {
      * @param password The password of the user
      * @param phone    The phone number of the user
      */
-    public User(String username, String email, String password, String phone) {
+    public User(String username, String email, String password, String phone, Long createdAt) {
```

Comment:
@Jayanta-Redapple Missing param in javadocs for `createdAt`. Lets keep our code documentation up-to-date along the way.